### PR TITLE
feat(sveltekit): Add feature selection

### PR DESCRIPTION
**Pull request by onurtemizkan**
Ref: #622

### CODE REVIEW

File: src/sveltekit/sdk-setup.ts [hunk 1]
```diff
@@ -50,6 +54,23 @@ export async function createOrMergeSvelteKitFiles(
...
+      prompt: `Do you want to enable ${chalk.bold(
+        'Tracing',
+      )} to track the performance of your application?`,
+      enabledHint: 'recommended',
+    },
+    {
+      id: 'replay',
+      prompt: `Do you want to enable ${chalk.bold(
+        'Sentry Session Replay',
+      )} to get reproduction of frontend errors via user sessions?`,
```
Comment (Lms24) [inline src/sveltekit/sdk-setup.ts:69]: The sentence reads funky to me and theoretically, replay can also surface backend errors. What about something like this?

```suggestion
      )} to get a video-like reproduction of errors during a user session?`,
```

I think we should also adjust this in the other wizards where we use `featureSelectionPrompt`.

**Review by Lms24 — COMMENTED** [review]
Sorry for the delayed review @onurtemizkan! I gave the wizard a try with the various permutations and found some things:

- if I select "No" for tracing but already have server or client hooks files prior to running the wizard, `tracesSampleRate` is still added.
- Likewise, replay is added if I select "No" if a client hooks file exists prior to running the wizard

If I didn't have hooks files prior to running the wizard, things worked correctly 👍

**Comment by Lms24** [comment]
heads-up: I'm just assigning myself to the PR to track internally that I'm reviewing it. Please continue as usual :)

**Comment by onurtemizkan** [comment]
@Lms24, thanks for the review! Updated the PR.

**Review by Lms24 — COMMENTED** [review]
Thanks for updating! Can confirm that the init calls now look correct for previously existing client and server hooks files. 

~One last thing: We should also not add the `replayIntegration` import if replay is not selected.~

**Review by Lms24 — APPROVED** [review]
Sorry, ignore my last review comment! I think the import was just there because I didn't reset the client hooks file correctly. LGTM!

**Pull request merged**